assets/scss/components/compat/woocommerce/_breadcrumbs.scss

```diff
@@ -26,4 +27,8 @@
 	display: flex;
 	font-size: 14px;
 	justify-content: space-between;
+
+	@media (min-width: #{$tablet-sm}) {
+		flex-direction: column;
+	}
```

Comment:
@preda-bogdan Let's move this in a separate mixin in the [media queries file](https://github.com/Codeinwp/neve/blob/cc2943adac7deceb856a2e1d3bd518fa2423de7f/assets/scss/components/compat/woocommerce/_media-queries.scss). I know this is tedious, but otherwise `media (min-width: #{$tablet-sm}) {}` will be added again in the final build of the file.

Reply:
Thank you for explaining this, I was not aware of this behavior. 🏎️